doc/getting_started.rst

```diff
@@ -64,7 +64,7 @@ Environment Variables
 WAM-Server needs at least the following environment variables:
 
 - CONFIG_PATH: Path to configuration file (mainly includes database configurations, see :ref:`configuration`)
-- DJANGO_SECRET_KEY: Set a secret key for *settings.py* (from *settings.py*: SECURITY WARNING: keep the secret key used in production secret!)
+- DJANGO_SECRET_KEY: Set a secret key for *settings.py* (SECURITY WARNING: keep the secret key used in production secret!)
 - WAM_APPS: Apps which shall be loaded within *INSTALLED_APPS*. Additionally, individual app settings are loaded (see :ref:`app_settings`).
```

Comment:
I think theses variables could be more descriptive in terms of the formatting of their possible values : 
as I understood it, `WAM_APPS` should contain the name of the folder in which each app is located separated by commas?

For `DJANGO_SECRET_KEY` : is the description "Set a secret key for *settings.py*" suppose to explain the purpose of this variable or it ask the user to set a secret key (could it be anything then?)

Reply:
Yes - `DJANGO_SECRET_KEY` can be any str, but should be a strong password - there is also a [django secret key generator](https://www.miniwebtool.com/django-secret-key-generator/)